modules/support_ticket/support_ticket.module

```diff
@@ -266,6 +266,54 @@ function support_ticket_load($nid = NULL, $reset = FALSE) {
 }
 
 /**
+ * Determines what fields have changed when the support ticket is updated.
+ *
+ * @param object $support_ticket
+ *   The support ticket entity object.
+ */
+function support_ticket_entity_update($support_ticket) {
+  $original = $support_ticket->original;
+  $fields_original = $support_ticket->original->getFields();
+  $fields_updated = $support_ticket->getFields();
+
+  $comment = '';
+
+  // Don't log changes to the following fields when the support_ticket is updated.
+  // @todo: "field_ticket_update" is a configurable comment field; I think we can simply
+  //        ignore any comment field.
+  $ignore = array('stid', 'uuid', 'vid', 'uid', 'langcode', 'changed', 'revision_timestamp', 'revision_uid', 'revision_log', 'revision_translation_affected', 'field_ticket_update');
+  foreach ($fields_updated as $name => $field) {
+    if (!in_array($name, $ignore)) {
+      $updated_value = $field->getValue();
+      $original_value = isset($fields_original[$name]) ? $fields_original[$name]->getValue() : FALSE;
+      foreach ($updated_value as $number => $values) {
+        foreach ($values as $key => $value) {
+          if ($original_value && isset($original_value[$number][$key]) && $value != $original_value[$number][$key]) {
+            $old_value = $original_value[$number][$key];
+            $comment .= "  <div class=\"metadata-change-$name\">$name: '$old_value' => '$value'</div>\n";
+          }
+        }
+      }
+    }
+  }
+
+  if (!empty($comment)) {
+    // @todo: need to set text format (configurable?)
```

Comment:
If it's always one user, it should be fine to show a list of text formats that particular user can access to make it configurable. Or maybe just the default text format for that user is fine? (They can change the default if they want...)